cvprac/cvp_api.py

```diff
@@ -361,6 +362,48 @@ def get_inventory(self, start=0, end=0, query=''):
                     dev['containerName'] = ''
             return data
 
+    def add_devices_to_undefined(self, device_ips, wait=True):
+        ''' Add the list of devices to the undefined container.
+
+            Args:
+                device_ips (list): ip addresses of devices to be added
+                wait (boolean): If True, wait up to 60 seconds for
+                    new device to appear in the undefined container
+        '''
+
+        self.log.debug('add_devices_to_undefined: called')
+        if self.clnt.apiversion is None:
+            self.get_cvp_info()
+        if self.clnt.apiversion == 'v1':
+            self.log.debug('v1 Inventory API Call')
+            self.log.warning(
+                'add_devices_to_undefined: not implemented for v1 APIs')
+        else:
+            self.log.debug('v2 Inventory API Call')
+            data = {'hosts': device_ips}
+            self.clnt.post('/inventory/devices', data=data,
+                           timeout=self.request_timeout)
+
+            if wait:
+                # It can take a few moments for devices to appear in inventory
+                dev = None
```

Comment:
This function take multiple device addresses to be added to undefined but only checks that one of them has been added during the wait?